src/screens/PrivateChat/ContactsPicker/index.js

```diff
@@ -0,0 +1,259 @@
+      showModal: '',
+    });
+  };
+
+  render() {
+    return (
+      <View style={styles.nationsScreenContainer}>
+        <BackgroundImage />
+        <FakeNavigationBar />
+        <ScreenTitle title={i18n.t('screens.contactsPicker.title')} />
+        <ScrollView>
+          <Selectize
+            ref={selectize => this.selectize = selectize}
+            chipStyle={styles.chip}
+            chipIconStyle={styles.chipIcon}
+            label='To:'
+            itemId='id'
+            filterOnKey='name'
+            items={this.props.contacts}
+            showItems='always'
+            error={this.state.addContactError}
+            listStyle={styles.list}
+            baseColor={Colors.BitnationLinkOrangeColor}
+            tintColor={Colors.BitnationLinkOrangeColor}
+            middleComponent={
+              <TouchableOpacity
+                  activeOpacity={0.6}
+                  onPress={this.addContact}
+                  style={styles.listRow}>
+                <View style={styles.listWrapper}>
+                  <View style={styles.listIcon}>
+                    <Image
+                      source={AssetsImage.avatarIcon}
+                      style={styles.avatarSmall}
+                    />
+                  </View>
+                  <View style={styles.listIcon}>
+                    <Text>
+                      {i18n.t('screens.contactsPicker.newContactFromClipboard')}
+                    </Text>
+                  </View>
+                </View>
+              </TouchableOpacity>
+            }
+            renderRow={(id, onPress, item) => (
+              <TouchableOpacity
+                  activeOpacity={0.6}
+                  key={id}
+                  onPress={onPress}
+                  style={styles.listRow}>
+                <View style={styles.listWrapper}>
+                  <View style={styles.listIcon}>
+                    <Image
+                      source={imageSource(item.profile.image) || AssetsImage.avatarIcon}
+                      style={styles.avatarSmall}
+                    />
+                  </View>
+                  <View style={styles.listIcon}>
+                    <Text style={styles.listPrimaryText}>{item.profile.name}</Text>
+                  </View>
+                </View>
+              </TouchableOpacity>
+            )}
+            renderChip={(id, onClose, item, style, iconStyle) => (
+              <Chip
+                key={id}
+                iconStyle={iconStyle}
+                onClose={onClose}
+                text={item.profile.name}
+                style={style}
+              />
+            )}
+          />
+        </ScrollView>
+
+        <InvalidKeyModal
+          done={this.dismissModal}
+          visible={this.state.showModal === INVALID_MODAL_KEY}
+        />
+
+        {this.state.loading === true && <Loading />}
+      </View>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  contacts: state.contacts.contacts.map(contact => ({
```

Comment:
I would better map contacts to the view required by `Selectize` right where you pass it to `Selectize` component. That way we can be sure that we have array of contacts (of type `Contact`) on the `ContactsPicker` component.